src/server/page_server.cpp

```diff
@@ -181,6 +185,14 @@ page_server::connection_handler::receive_stop_log_prior_dispatch (tran_server_co
   m_conn->respond (std::move (a_sp));
 }
 
+void
+page_server::connection_handler::receive_oldest_active_mvccid (tran_server_conn_t::sequenced_payload &a_sp)
+{
+  /* TODO dummy function. will be modified corretly. soon  */
```

Comment:
maybe, `assert (m_server_type == transaction_server_type::PASSIVE);` ?

Reply:
Opps, already merged. I'll consider this a following PR, maybe in http://jira.cubrid.org/browse/LETS-548.